packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -229,30 +235,28 @@ export const ChatScreen = () => {
     }
   }, [earliestUnreadIndex, chatMessages])
 
-  const handleScrollToIndexFailed = useCallback(
-    (e) => {
-      setTimeout(() => {
-        flatListRef.current?.scrollToIndex({
-          index: e.index,
-          viewPosition: 0.95,
-          animated: false
-        })
-      }, 10)
-    },
-    [flatListRef]
-  )
+  const handleScrollToIndexFailed = useCallback((e) => {
+    setTimeout(() => {
+      flatListRef.current?.scrollToIndex({
+        index: e.index,
+        viewPosition: 0.95,
+        animated: false
+      })
+    }, 10)
+  }, [])
 
-  const handleScrollToTop = () => {
-    if (
-      chatId &&
-      chat?.messagesStatus !== Status.LOADING &&
-      chat?.messagesSummary &&
-      chat?.messagesSummary.prev_count > 0
-    ) {
+  const shouldFetchMoreMessages =
+    chatId &&
+    chat?.messagesStatus !== Status.LOADING &&
+    chat?.messagesSummary &&
```

Comment:
nit: can probably combine the 2 lines into `chat?.messagesSummary?.prev_count > 0`

Reply:
Would look like `(chat?.messagesSummary?.prev_count ?? 0) > 0`. I think what we have here is cleaner but lmk @sddioulde .